src/pages/walled-gardens.md

```diff
@@ -3,17 +3,162 @@ title: 'Bringing Competition to Walled Gardens'
 permalink: '/walled-gardens-report/'
 layout: 'layouts/paper.njk'
 metaDesc: 'The full Bringing Competition to Walled Gardens report, published by Open Web Advocacy.'
-subtitle: 'Third Party Browsers & Web Apps - VERSION 1.0'
-paperName: 'OWA - Bringing Competition to Walled Gardens - v1.0'
+subtitle: 'Third-Party Browsers & Web Apps - VERSION 1.2'
+paperName: 'OWA - Bringing Competition to Walled Gardens - v1.2'
 paperSize: '9.2MB'
+pageNonCriticalScripts: [ '/js/walled-gardens/collapser.js' ]
 ---
 
+<style>
```

Comment:
Addressed in 5ff20c8

Note: Regarding the js, I've just moved the `pageNonCriticalScripts` front matter to `_includes/layout/paper.njk` as suggested above, so it works the same.

If we want to inline the script, I see two aproaches:
* just move the contents of `collapser.js` into a `<script>` tag in `paper.njk`, remove the `pageNonCriticalScripts` functionality
* Keep the `pageNonCriticalScripts` functionality but change the code in `base.njk` to `{% include %}` the js instead of linking to it